File: tests/generators/ios/objc/files/EntityMapping.m
  RKEntityMapping *postResponseMapping = [RKEntityMapping mappingForEntityForName:@"PostResponse" inManagedObjectStore:managedObjectStore];
  postResponseMapping.identificationAttributes = @[@"theID"];
  [postResponseMapping addAttributeMappingsFromDictionary:@{@"body": @"body", @"id": @"theID", @"title": @"title"}];

File: tests/generators/ios/swift/files/EntityMapping.swift
    let postResponseMapping = RKEntityMapping(forEntityForName: "PostResponse", inManagedObjectStore: managedObjectStore)
    postResponseMapping.identificationAttributes = ["theID"]
    postResponseMapping.addAttributeMappingsFromDictionary([ "body": "body", "id": "theID", "title": "title" ])
